R/guessColnames.R

```diff
@@ -20,6 +20,8 @@ guessColnames <- function(x, m) {
       colnames(x)[j] <- "var"
     } else if (all(x[[j]] %in% readSymbol(m, "hs"))) {
       colnames(x)[j] <- "hs"
+    } else if (all(x[[j]] %in% readSymbol(m, "var"))) {
+      colnames(x)[j] <- "var"
```

Comment:
Exactly this code snippet also exists four lines above - either adding this does not solve your issues or there was no issue, at least not in the trunk. Or something weird is happening here ...

Reply:
Thanks for spotting. I overlooked this and tried to fix the bug by adding these lines to then find `guessColnames` was not working properly due to the gamstransfer update. Never realised, I duplicated lines here. I removed them.